### PR TITLE
Events page

### PR DESCRIPTION
**Pull request by ghost**
events page

### CODE REVIEW

**Comment by ghost** [comment]
Using the LSCC group for the request, when no meetup exists we can display a message:

events page - no meetups

File: assets/js/events.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	     url : apiEndpoint,
+	     success: function(data){
+				$(data.data).each(function(i, lscc_event){
+					lscc_event.time = new Date(lscc_event.time).toDateString();
+					lscc_event.description = lscc_event.plain_text_description.substring(0,300) + "...";
+					var eventHtml = template(lscc_event);
+					$("#lssc_events").append($textElement.html(eventHtml).text());
+				});
+				if($(data.data)){
+					$("#lssc_events").html("<li><h2>We're still planning a Meetup</h2></li>");
```
Comment (dhatanian) [inline assets/js/events.js]: How about an invitation to come back soon?

> We are still planning the next meetup, come back soon for more!

Comment (ghost) [reply assets/js/events.js]: I changed the message. Thanks.

**Comment by jukecraft** [comment]
Consider replacing the stretched GDCR picture with a logo from here:
http://globalday.coderetreat.org/

There is a logo here:
https://drive.google.com/drive/u/0/folders/0BzgSBslYeykbaXgycV9QMUpDMjQ

**Comment by ghost** [comment]
events page-es

**Pull request merged**